set.mm

```diff
@@ -265985,8 +266433,11 @@ U C_ ( N ` { X } ) ) -> ( U = ( N ` { X } ) \/ U = { .0. } ) ) $=
                                      ( ( subringAlg ` w ) ` ( Base ` w ) ) ) $.
 
     $( Define the class of left ideals of a given ring.  An ideal is a
-       submodule of the ring viewed as a module over itself.  (Contributed by
-       Stefan O'Rear, 31-Mar-2015.) $)
+       submodule of the ring viewed as a module over itself.  This definition
+       corresponds to the usual textbook definition of a (left) ideal of a ring
+       to be a subgroup of the additive group of the ring which is closed under
+       left-multiplication by elements of the full ring, see ~ dflidl2 .
+       (Contributed by Stefan O'Rear, 31-Mar-2015.) $)
     df-lidl $a |- LIdeal = ( LSubSp o. ringLMod ) $.
```

Comment:
This sentence applies to `~ dflifl2`, but not to `df-lidl`, so I don't think you cant write "_This definition corresponds to_..." : Here I'd rather say something like "_**For the** usual textbook definition ..., see_ `~dflidl2`"

Reply:
Yes, I have modified the comment accordingly.